classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize
+    @label = ''
+    @students = []
+  end
```

Comment:
- [ ] Great bob 👏🏻  Let's modify our constructor to accept parameters and use them. I might be like this:

```
class Animal
  def initialize(name, number_of_legs)
    @id = Random.rand(1..1000)
    @name = name
    @number_of_legs = number_of_legs
  end
end
```

Visit this [link](`https://github.com/rbanos-mv/oop_school_library/pull/3/files#diff-bd6d401bca7b47b397906cb44f68b377a8e8570fed0f0d54848131cf14baa70d`)